packages/financial-templates-lib/src/price-feed/StLouisFedGovPriceFeed.ts

```diff
@@ -45,6 +48,14 @@ export class StLouisFedGovPriceFeed extends PriceFeedInterface {
       return Web3.utils.toBN(parseFixed(number.toString().substring(0, priceFeedDecimals), priceFeedDecimals).toString());
     };
   }
+  public getTimestampInTimeZone = (timeZone: string) => {
+    const date = new Date();
+    const options = {
+      timeZone: timeZone,
+    };
+    const timestamp = date.toLocaleString('en-US', options);
+    return Math.round(new Date(timestamp).getTime() / 1000);
+  };
 
   public async update(ancillaryData?: string): Promise<void> {
     const currentTime = await this.getTime();
```

Comment:
this would be a more appropriate place to get the current time depending on boolean `this.useStLouisLocalTime`